build.zig

```diff
@@ -124,7 +124,17 @@ pub fn build(b: *zbs.Builder) !void {
         rivertile.install();
     }
 
-    b.installFile("protocol/river-layout-v3.xml", "share/river/river-layout-v3.xml");
+    {
+        const file = try fs.path.join(b.allocator, &[_][]const u8{ b.cache_root, "river-protocols.pc" });
+        const pkgconfig_file = try std.fs.cwd().createFile(file, .{});
+
+        const writer = pkgconfig_file.writer();
+        try writePkgconfigFile(writer, b.install_prefix, full_version);
+        defer pkgconfig_file.close();
+
+        b.installFile("protocol/river-layout-v3.xml", "share/river-protocols/staging/river-layout-v3.xml");
```

Comment:
After thinking about this a bit more, I don't think I want to make the distinction between stable/staging protocols. I never intend to say that river will avoid all breaking changes until the end of time, and the same goes for river's protocols. Things will get highly stable eventually, but set in stone with no chance to fix mistakes.

Reply:
So you want a `stable` folder or just no folder, everything in river-protocols?

Reply:
Just no folder, everything in `share/river-protocols`